charts/incubator/lancache-monolithic/values.yaml

```diff
@@ -1,7 +1,7 @@
 image:
-  repository: tccr.io/truecharts/lancache-monolithic
+  repository: lancachenet/monolithic
   pullPolicy: IfNotPresent
-  tag: latest@sha256:ed47044d1421424a134ff80e38da33d4a362812364e99dc1a09521a55dab42c1
+  tag: latest
```

Comment:
Why is that changed?

Reply:
I not sure if this is the latest lancache build thats why i replaced it

Reply:
You can check the source of our mirror in `truecharts/containers` repo. It auto updates every day

Reply:
This needs to changed back to what it was.